ckanext/scheming/plugins.py

```diff
@@ -449,5 +500,6 @@ def _expand_schemas(schemas):
             if fname not in s:
                 continue
             s[fname] = [_expand_preset(f) for f in s[fname]]
+            s[fname] = [_expand_vocabulary(f) for f in s[fname]]
```

Comment:
This means vocabulary choices are expanded into the schema once on startup, so the server needs to be restarted if the vocabulary tags are modified?